resources/assets/js/Pages/Calendar.js

```diff
@@ -59,35 +60,41 @@ export default class Calendar extends Component {
   }
 
   render() {
-    return (
-      [this.renderDays(),
-      this.state.start ? <div key="calendar-grid" className="calendar-grid">{this.renderWeeks()}</div> : '',
-      <div key="calendar-manage" className="calendar-manage">
-        {
-          this.props.isTutor ?
-          <div className="calendar-setting" onClick={(_)=>this.setState({ addLesson: true })}>
-            <img src="/images/icons8-plus-50.png" /> Add Lesson
+    return ([
+      <Sidebar selected={this.props.page} isTutor={this.props.isTutor} />,
+      <div className="width-fill flex-rows">
+        <div className="calendar-container">
+          {this.renderDays()}
+          {this.state.start ? <div key="calendar-grid" className="calendar-grid">{this.renderWeeks()}</div> : ''}
+          <div key="calendar-manage" className="calendar-manage">
+            {
+              this.props.isTutor ?
+              <div className="calendar-setting" onClick={(_)=>this.setState({ addLesson: true })}>
```

Comment:
why dont you put the setState in a separate function and just call that function for onClick? This avoids the lambda without a reference as well